libs/filamat/src/MaterialBuilder.cpp

```diff
@@ -337,6 +336,16 @@ static void showErrorMessage(const char* materialName, uint8_t variant,
 }
 
 Package MaterialBuilder::build() noexcept {
+    GLSLTools::init();
```

Comment:
Can init() be called several times safely? I don't remember what it does. I think it's safe but we should make sure… :)

Reply:
I tested and it appears that multiple calls to `init` seems to be fine, but looking at [glslang's source](https://github.com/google/filament/blob/0548b285357453def900351c60e5e87e2082cc32/third_party/glslang/glslang/MachineIndependent/ShaderLang.cpp#L1258), it might be better to restrict ourselves to calling `InitializeProcess` only once.

Reply:
Agreed.